docs/lava-job-priority.md

```diff
@@ -19,12 +19,14 @@ requirements, and it fully takes advantage of having 101 priorities available.
   - stable-rc sanity jobs (all branches)
   - mainline sanity jobs
 - 79: [Developer builder](developer-builder.md) jobs
-- 78: stable-rc-4.4
-- 77: stable-rc-4.9
-- 76: stable-rc-4.14
+- 72: stable-rc-4.4
+- 73: stable-rc-4.9
+- 74: stable-rc-4.14
 - 75: stable-rc-4.19
-- 74: stable-rc 5.4
-- 73: reserved for 2020 LTS kernel (perhaps 5.9?)
+- 77: stable-rc 5.4
```

Comment:
wouldn't it make more sense for 5.4 to be 76?

Reply:
@danrue  Please check latest changes as per your comments.